### PR TITLE
Abdul/app service shared notepad

### PR DESCRIPTION
**Pull request by v-abdrahman**
Created a folder for AppServiceSharedNotepad app & added all the project files for windows 10 Iot Enterprise.

Following changes are done:

1) file modified : NotepadClientBackgroundApplication.csproj
    TargetPlatformVersion from 10.0.17763.0 to 10.0.19041.0 and TargetPlatformMinVersion from 10.0.10240.0 to 10.0.15063.0.

2) file modified : NotepadService.csproj
    TargetPlatformVersion from 10.0.17763.0 to 10.0.19041.0 and TargetPlatformMinVersion from 10.0.10240.0 to 10.0.15063.0.

3) file modified : NotepadServiceClientApp.csproj
    TargetPlatformVersion from 10.0.17763.0 to 10.0.19041.0 and TargetPlatformMinVersion from 10.0.10240.0 to 10.0.15063.0.

2) file modified : README.md
    Adding windows 10 iot Enterprise in products.
    Updated the Windows Platform from IoT Core to IoT Enterprise
    Added the BIOS setting for UPBOARD Pin configuration
    Added the app *packages installing steps

### CODE REVIEW

File: AppServiceSharedNotepad/CS/NotepadClientBackgroundApplication/NotepadClientBackgroundApplication.csproj [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
+    <TargetPlatformVersion>10.0.19041.0</TargetPlatformVersion>
+    <TargetPlatformMinVersion>10.0.15063.0</TargetPlatformMinVersion>
+    <MinimumVisualStudioVersion>14</MinimumVisualStudioVersion>
+    <EnableDotNetNativeCompatibleProfile>true</EnableDotNetNativeCompatibleProfile>
+    <FileAlignment>512</FileAlignment>
+    <ProjectTypeGuids>{A5A43C5B-DE2A-4C0C-9213-0A381AF9435A};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
+    <AllowCrossPlatformRetargeting>false</AllowCrossPlatformRetargeting>
+    <PackageCertificateKeyFile>NotepadClientBackgroundApplication_TemporaryKey.pfx</PackageCertificateKeyFile>
```
Comment (neilsh-msft) [inline AppServiceSharedNotepad/CS/NotepadClientBackgroundApplication/NotepadClientBackgroundApplication.csproj:21]: NotepadClientBackgroundApplication_TemporaryKey.pfx file is missing from pull request.

File: AppServiceSharedNotepad/CS/NotepadService/NotepadService.csproj [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
+    <TargetPlatformVersion>10.0.19041.0</TargetPlatformVersion>
+    <TargetPlatformMinVersion>10.0.15063.0</TargetPlatformMinVersion>
+    <MinimumVisualStudioVersion>14</MinimumVisualStudioVersion>
+    <EnableDotNetNativeCompatibleProfile>true</EnableDotNetNativeCompatibleProfile>
+    <FileAlignment>512</FileAlignment>
+    <ProjectTypeGuids>{A5A43C5B-DE2A-4C0C-9213-0A381AF9435A};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
+    <AllowCrossPlatformRetargeting>false</AllowCrossPlatformRetargeting>
+    <PackageCertificateKeyFile>NotepadService_TemporaryKey.pfx</PackageCertificateKeyFile>
```
Comment (neilsh-msft) [inline AppServiceSharedNotepad/CS/NotepadService/NotepadService.csproj:21]: NotepadService_TemporaryKey.pfx file is missing from pull request.

File: AppServiceSharedNotepad/CS/NotepadServiceClientApp/NotepadServiceClientApp.csproj [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+    <RootNamespace>NotepadServiceClientApp</RootNamespace>
+    <AssemblyName>NotepadServiceClientApp</AssemblyName>
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
+    <TargetPlatformVersion>10.0.19041.0</TargetPlatformVersion>
+    <TargetPlatformMinVersion>10.0.15063.0</TargetPlatformMinVersion>
+    <MinimumVisualStudioVersion>14</MinimumVisualStudioVersion>
+    <FileAlignment>512</FileAlignment>
+    <ProjectTypeGuids>{A5A43C5B-DE2A-4C0C-9213-0A381AF9435A};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
+    <PackageCertificateKeyFile>NotepadServiceClientApp_TemporaryKey.pfx</PackageCertificateKeyFile>
```
Comment (neilsh-msft) [inline AppServiceSharedNotepad/CS/NotepadServiceClientApp/NotepadServiceClientApp.csproj:19]: NotepadServiceClientApp_TemporaryKey.pfx file is missing from pull request.

File: AppServiceSharedNotepad/CS/README.md [hunk 4]
```diff
@@ -0,0 +1,322 @@
...
+We’ll create a simple notepad [app service](https://docs.microsoft.com/en-us/windows/uwp/launch-resume/how-to-create-and-consume-an-app-service).  We'll also create two simple app service clients. One that writes to the notepad service and one that reads from the notepad service.
+
+## Load the projects in Visual Studio
+___
+
+* You can find the source code for this sample by downloading a zip of all of our samples in WIndows 10 IoT Enterprise folder and navigating to the `samples-develop\AppServiceBlinky`. 
+* Make a copy of the folder on your disk and open the projects from Visual Studio. 
+* NotepadService.sln implements the app service and must be started first.
+* NotepadClientBackgroundApplication.sln implements a client that writes messages to the app service.
+* NotepadServiceClientApp.sln implements a windows that retrieves messages from the client and displays them.
```
Comment (neilsh-msft) [inline AppServiceSharedNotepad/CS/README.md]: should be "implements a window"  not "Implements a windows"

File: AppServiceSharedNotepad/CS/README.md [hunk 5]
```diff
@@ -0,0 +1,322 @@
...
+* Make a copy of the folder on your disk and open the projects from Visual Studio. 
+* NotepadService.sln implements the app service and must be started first. 
+* NotepadClientBackgroundApplication.sln implements the app service client.
+
+## Deploy your app
+
+* * *
+
+With the application open in Visual Studio 2019, set the architecture in the toolbar dropdown. If you’re building for UpBaord, select `x64`.
```
Comment (neilsh-msft) [inline AppServiceSharedNotepad/CS/README.md]: Misspelled "UpBaord", please use "UP Board" across all files.

File: AppServiceSharedNotepad/CS/README.md [hunk 6]
```diff
@@ -0,0 +1,322 @@
...
+
+When everything is set up, you should be able to press F5 from each instance of Visual Studio. 
+
+The NotepadService app will deploy and start on the Windows IoT device, and you should see the package family name printed the debug output window on Visual Studio.  
+
+Before pressing F5 for NotepadClientBackgroundApplication and NotepadServiceClientApp projects verify that the value of connection.PackageFamilyName in StartupTask.cs and MainPage.xaml.cs matches the value output in the output window by NotepadService.
+
+If it doesnot any value output, You will get the value by marking the breakpoint in debug mode for
```
Comment (neilsh-msft) [inline AppServiceSharedNotepad/CS/README.md]: "If it doesnot any value output, You will get", does not make sense.   Should it be "If it does not have any output value, you will get" ?

**Review by neilsh-msft — CHANGES_REQUESTED** [review]

File: README.md [hunk 7]
```diff
@@ -1,74 +1,319 @@
...
-
-## Samples by category
-
-### Azure IoT Edge
-
-| Name           | Description      |  
-|----------------|------------------|  
-| [interop-textmsg-consoleapp](https://aka.ms/WinIoTSamples-interop-textmsg-consoleapp) | Basic interop sample demonstrating text messaging between a Windows console app and an Edge module. |
-| [interop-customvision-textmsg-uwpapp](https://aka.ms/WinIoTSamples-interop-customvision-textmsg-uwpapp) | <p>Two more advanced interop samples which demonstrate bidirectional communication between a Windows application and an Edge module. </p><ul><li>Text messaging between a UWP application and an Edge module. </li><li>A 'Custom vision' machine learning interop sample with a fruit classifier which uses a Windows UWP app to send camera frames to an Edge module for identification.</li></ul>|  
+---
```
Comment (rsameser) [inline README.md]: Do **NOT** update the README for the repo please

**Review by rsameser — CHANGES_REQUESTED** [review]
Please address comments - and do not update repo's ReadMe page

File: AppServiceSharedNotepad/CS/NotepadClientBackgroundApplication/NotepadClientBackgroundApplication.csproj (hunk 1 above)
Comment (v-abdrahman) [reply AppServiceSharedNotepad/CS/NotepadClientBackgroundApplication/NotepadClientBackgroundApplication.csproj:21]: Included the Temporary Key file

File: AppServiceSharedNotepad/CS/NotepadService/NotepadService.csproj (hunk 2 above)
Comment (v-abdrahman) [reply AppServiceSharedNotepad/CS/NotepadService/NotepadService.csproj:21]: Included the Temporary Key file

File: AppServiceSharedNotepad/CS/NotepadServiceClientApp/NotepadServiceClientApp.csproj (hunk 3 above)
Comment (v-abdrahman) [reply AppServiceSharedNotepad/CS/NotepadServiceClientApp/NotepadServiceClientApp.csproj:19]: Included the Temporary Key file

File: README.md (hunk 7 above)
Comment (v-abdrahman) [reply README.md]: I will not update the readme for the repository.

File: AppServiceSharedNotepad/CS/README.md (hunk 4 above)
Comment (v-abdrahman) [reply AppServiceSharedNotepad/CS/README.md]: Updated it.

File: AppServiceSharedNotepad/CS/README.md (hunk 5 above)
Comment (v-abdrahman) [reply AppServiceSharedNotepad/CS/README.md]: Updated it

File: AppServiceSharedNotepad/CS/README.md (hunk 6 above)
Comment (v-abdrahman) [reply AppServiceSharedNotepad/CS/README.md]: Updated it

File: README.md (hunk 7 above)
Comment (rsameser) [reply README.md]: Please remove this file!

Comment (v-abdrahman) [reply README.md]: I have removed the file

Comment (v-abdrahman) [reply README.md]: Its done.

**Pull request closed without merging**